webapp/portlet/src/main/webapp/vue-apps/space-infos-app/components/ExoSpaceInfos.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-app>
-    <div class="body-1 text-uppercase text-sub-title">{{ $t("social.space.description.title") }}</div>
+    <div class="body-1 text-uppercase text-sub-title mx-n1">{{ $t("social.space.description.title") }}</div>
```

Comment:
Could you check where the margin is added and fix it from there instead of adding a negative margin ?